src/definitions/elements/label.less

```diff
@@ -1181,7 +1180,7 @@ a.ui.basic.label:hover {
 .ui.pointing.label:before {
   background-color: inherit;
   background-image: inherit;
-  border-width: none;
+  border-width: 0px;
```

Comment:
Could you change the likes of these to be `0` instead of having a floating `px` at the end

Reply:
Just this one or all of them ?

Reply:
All of them, if you want you could do it for all files.